Document FIPS limitation for buckets with dot (#26436)
DataDog/documentation: content/en/logs/log_configuration/archives.md

Go into your [AWS console][1] and [create an S3 bucket][2] to send your archives to.

{{< site-region region="gov" >}}

> <div class="alert alert-warning"> Datadog Archives do not support bucket names with dots (.) when integrated with an S3 FIPS endpoint which relies on virtual-host style addressing. Learn more from AWS documentation. <a href="https://aws.amazon.com/compliance/fips/">AWS FIPS</a> and <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html">AWS Virtual Hosting</a>.</div>

Reviewer: ```suggestion
<div class="alert alert-warning"> Datadog Archives do not support bucket names with dots (.) when integrated with an S3 FIPS endpoint which relies on virtual-host style addressing. For more information, see the AWS <a href="https://aws.amazon.com/compliance/fips/">AWS FIPS</a> and <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html">AWS Virtual Hosting</a> documentation.</div>
```
